src/components/NavBar.vue

```diff
@@ -1,73 +1,84 @@
         </template>
         <template #end>
-          <div class="hidden md:flex items-center space-x-4"> 
-            <template v-for="route in endRoutes" :key="route.path">
-              <button-on
-                :class="{'font-bold': isActive(route.path)}"
-                class="p-2 mx-4 text-sm transition duration-300 transform hover:-translate-y-1 hover:bg-gray-700"
-                :label="route.label"
-                :icon="route.icon"
-                @click="navigateTo(route.path)" />
-            </template>
-          </div>
- 
-          <div class="md:hidden relative">
-            <button-on @click="toggleSideBar" class="focus:outline-none"> 
-              <i :class="['pi', sidebarVisible ? 'pi-times' : 'pi-bars']" class="text-lg"></i>
+          <section class="hidden md:flex">
+            <button-on 
+              label="" 
+              @click="isAboutVisible = true" 
+              icon="pi pi-info-circle"
+              class="p-2 mx-4 text-sm transition duration-300 transform hover:-translate-y-1 hover:bg-gray-700"
+              title="Sobre" 
+              severity="warn" 
+            />
+          </section>
+          <span class="md:hidden relative">
+            <button-on @click="toggleSideBar" class="focus:outline-none">
+              <i :class="['pi', isSidebarVisible ? 'pi-times' : 'pi-bars']" class="text-lg"></i>
             </button-on>
-          </div>
+          </span>
         </template>
       </tool-bar>
-    </div>
- 
-    <div v-if="sidebarVisible" class="fixed top-0 right-0 z-50 flex flex-col bg-gray-800 text-white w-44 max-h-screen overflow-y-auto p-5">
-      <div class="flex justify-end">
+    </section>
+
+    <section v-if="isSidebarVisible"
+      class="fixed top-0 right-0 z-50 flex flex-col bg-gray-800 text-white w-44 max-h-screen overflow-y-auto p-5">
+      <span class="flex justify-end">
         <button @click="toggleSideBar" class="focus:outline-none">
           <i class="pi pi-times"></i>
         </button>
-      </div>
-      <div class="space-y-4 mt-8">
+      </span>
+      <section class="space-y-4 mt-8">
         <template v-for="route in mainRoutes" :key="route.path">
-          <button-on
-            :class="{'font-bold': isActive(route.path)}"
+          <button-on 
+            :class="{ 'font-bold': isActive(route.path) }"
             class="p-2 text-sm transition duration-300 transform hover:bg-gray-700 cursor-pointer flex items-center space-x-2"
-            :label="route.label"
-            :icon="route.icon"
-            @click="navigateTo(route.path)" />
+            :title="route.label" 
+            :label="route.label" 
+            :icon="route.icon" 
+            @click="navigateTo(route.path)" 
+          />
         </template>
-        <template v-for="route in endRoutes" :key="route.path">
-          <button-on
-            :class="{'font-bold': isActive(route.path)}"
+        <section>
+          <button-on 
+            label="" 
```

Comment:
Try to put a label here for the "About", when it's mobile

Reply:
Thanks! I'll review it!